games/l.yaml

```
@@ -82,7 +82,7 @@
   development: active
```

```suggestion
  development: halted
```
let's update this as well; repo is archived